lib/is_it_working/handler/basic_auth.rb

```diff
@@ -0,0 +1,39 @@
+begin
+  require 'rack'
+rescue NameError
+  raise NameError, 'IsItWorking::Handler::BasicAuth requires rack, please add it to your Gemfile.'
+end
```

Comment:
Kind of feels like we should add `rack` as a dep by moving this up? https://github.com/customink/is_it_working/blob/master/is_it_working.gemspec#L30